cli/commands/hclfmt/command.go

```diff
@@ -9,9 +9,10 @@ import (
 const (
 	CommandName = "hclfmt"
 
-	FlagNameTerragruntHCLFmt = "terragrunt-hclfmt-file"
-	FlagNameTerragruntCheck  = "terragrunt-check"
-	FlagNameTerragruntDiff   = "terragrunt-diff"
+	FlagNameTerragruntHCLFmt      = "terragrunt-hclfmt-file"
+	FlagNameTerragruntCheck       = "terragrunt-check"
+	FlagNameTerragruntDiff        = "terragrunt-diff"
+	FlagNameTerragruntHCLFmtStdin = "terragrunt-hclfmt-stdin"
```

Comment:
Missing integration test to validate end-to-end usage of `--terragrunt-hclfmt-stdin` flag, for example I commented lines 37-42 and all tests pass...

Examples:
https://github.com/gruntwork-io/terragrunt/blob/main/test/integration_download_test.go#L260
https://github.com/gruntwork-io/terragrunt/blob/main/test/integration_test.go

Reply:
Hi @denis256, do you have any ideas how to write a test like `cat terragrunt.hcl | terragrun hclfmt --terragrunt-hclfmt-stdin` in a platform agnostic way? 

All current tests are executing `terragrunt` with `App.Run` and I struggle to understand how to run the test without modifying a lot of code. Any hints?

Thank you!

Reply:
```
	input := "<contents of terragrunt.hcl>"
	os.Stdin = strings.NewReader(input)
```

Reply:
@denis256 thank you for the tip! 
I have added integration test.